src/sentry/snuba/metrics/extraction.py

```diff
@@ -73,7 +77,7 @@ class OnDemandMetricSpecVersioning:
     def get_query_spec_version(cls: Any, organization_id: int) -> SpecVersion:
         """Return spec version based on feature flag enabled for an organization."""
         org = Organization.objects.get_from_cache(id=organization_id)
-        if features.has("organizations:on-demand-metrics-query-spec-version-two", org):
+        if features.has(NEXT_SPEC_FEAT_FLAG, org):
             return cls.spec_versions[1]
         return cls.spec_versions[0]
```

Comment:
It's a little odd that we're making an array of spec versions then just using "this spec version" and "next spec version", we probably could have made this simpler? it's fine for now though I guess.